packages/wonder-blocks-toolbar/src/components/__docs__/toolbar.stories.js

```diff
@@ -0,0 +1,199 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import Color from "@khanacademy/wonder-blocks-color";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+import Toolbar from "@khanacademy/wonder-blocks-toolbar";
+
+import type {StoryComponentType} from "@storybook/react";
+import ComponentInfo from "../../../../../.storybook/components/component-info.js";
+import {name, version} from "../../../package.json";
+import ToolbarArgtypes, {
+    leftContentMappings,
+    rightContentMappings,
+} from "./toolbar.argtypes.js";
+
+export default {
+    title: "Toolbar / Toolbar",
+    component: Toolbar,
+    argTypes: ToolbarArgtypes,
+    parameters: {
+        componentSubtitle: ((
+            <ComponentInfo name={name} version={version} />
+        ): any),
+        docs: {
+            description: {
+                component: null,
+            },
+            source: {
+                // See https://github.com/storybookjs/storybook/issues/12596
+                excludeDecorators: true,
+            },
+        },
+    },
+    decorators: [
+        (Story: any): React.Element<typeof View> => (
+            <View style={styles.example}>{Story()}</View>
+        ),
+    ],
+};
+
+const styles = StyleSheet.create({
+    example: {
+        padding: Spacing.large_24,
+        alignItems: "center",
+        justifyContent: "center",
+    },
+    darkContainer: {
+        backgroundColor: "#0a2a66",
```

Comment:
Is this supposed to be the old dark blue? I have a feeling the background color was written in the .md file to match our old dark blue but then we updated `Color.darkBlue`. Maybe we should update this to use `Color.darkBlue` too?

Also, would it make sense to use `background` under parameters instead?

```
Dark.parameters = {
    backgrounds: {
        default: "darkBlue",
    },
}
```

Reply:
hmmm yeah good point! I'll use `darkBlue` instead + include the light button to illustrate better this example.

Reply:
@nishasy I updated the dark example following your suggestions. wdyt?

https://www.chromatic.com/test?appId=5e1bf4b385e3fb0020b7073c&id=636a90f833a0e76923978f37